version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.8.1-SNAPSHOT"
+version in ThisBuild := "0.9.1-SNAPSHOT"
```

Comment:
0.9.0-SNAPSHOT if we are to force a minor bump.

Reply:
I'll revert this to its previous state. I wrongly assumed it would be insync with the scalapb compilerplugin release.